WalletWasabi.Tests/RegressionTests/BuildTests.cs

```diff
@@ -354,7 +354,7 @@ public async Task BuildTransactionReorgsTestAsync()
 			// Spend the inputs of the tx so we know
 			var success = bitcoinStore.TransactionStore.TryGetTransaction(fundingTxId, out var invalidSmartTransaction);
 			Assert.True(success);
-			var invalidCoin = Assert.Single(((CoinsRegistry)wallet.Coins).AsAllCoinsView().CreatedBy(invalidSmartTransaction!.GetHash()));
+			var invalidCoin = Assert.Single(wallet.Coins.AsAllCoinsView().CreatedBy(invalidSmartTransaction!.GetHash()));
```

Comment:
Because `ICoinsView` interface is implemented not just by `CoinsRegistry` but by `CoinsView` as well, and those new methods are irrelevant for `CoinsView`.